docs/docs/installation_guide/installation/cloud9install.md

```diff
@@ -49,7 +49,7 @@ For more information on partitioning, read [Moving an environment and resizing/e
 
 ### Install Packer
 
-Packer is used to build AMIs. For steps on packer installation, refer to the [README](https://github.com/awslabs/service-workbench-on-aws/blob/b20208099d5acf51816ee4efd5b5bb3bf6d22fc8/addons/addon-base-raas/packages/serverless-packer/README.md).
+Packer is used to build AMIs. For steps on packer installation, refer to the [README](https://github.com/awslabs/service-workbench-on-aws/blob/b20208099d5acf51816ee4efd5b5bb3bf6d22fc8/addons/addon-base-raas/packages/serverless-packer/README.md#topics).
```

Comment:
Please direct to mainline branch of the documentation: https://github.com/awslabs/service-workbench-on-aws/blob/mainline/addons/addon-base-raas/packages/serverless-packer/README.md#topics.

Yes, we had the old commit hash in the URL, but this will improve the documentation as well!